src/ObjectUtils/ElfFile.cpp

```diff
@@ -120,6 +124,15 @@ class ElfFileImpl : public ElfFile {
   std::vector<orbit_grpc_protos::ModuleInfo::ObjectSegment> loadable_segments_;
 };
 
+bool IsHotpatchable(const absl::flat_hash_set<uint64_t>& hotpachable_addresses,
+                    uint64_t symbol_address) {
+  // The hotpatchable addresses stored in the elf file point to the first byte of the padding. We
+  // require the binary to be compiled with a five byte padding and a two byte nop at the function
+  // entry. So we check for "address - 5" to be listed as hotpatchable.
+  constexpr int kPaddingSize = 5;
```

Comment:
Will this be the same for COFF files? (Assuming we want to support that at some point.) Mainly wondering if we should have more info in the proto than just a boolean (though, arguably more data can also be added later.)

Reply:
Ok, thanks.